stackrox/stackrox
# ROX-25385: Display NVD CVSS in Vulnerability Reports

### Description

Frontend counterpart to #12809

Pending confirmation from Mansur, I added:

**Optional columns**

Include NVD CVSS 

### User-facing documentation


### Testing and quality


#### Automated testing


#### How I validated my change

Prerequisite before `npm run deploy` in ui folder

```sh
export ROX_NVD_CVSS_UI=true
```

1. `npm run lint` in ui/apps/platform
2. `npm run build` in ui/apps/platform
3. `npm run start` in ui/apps/platform

#### Manual testing

1. Visit /main/vulnerabilities/reports?action=create

    * Before changes, see absence of checkbox
        ![Configure_report_parameters_absence](https://github.com/user-attachments/assets/20b33de5-e5e2-43b4-a000-331833f28e3a)

    * After changes, see presence of checkbox
        ![Configure_report_parameters_presence](https://github.com/user-attachments/assets/7699b08a-d332-44b3-9688-920158f1ce00)

2. **Review and create**

    * Before changes, see absence of checkbox
        ![Review_and_create_absence](https://github.com/user-attachments/assets/4fe82db7-1cc6-401d-9ca1-b62e80dffac3)

    * After changes, see presence of checkbox
        ![Review_and_create_presence](https://github.com/user-attachments/assets/3a66110c-92db-4614-b6af-40121a82a695)

3. Visit report configuration

    * After changes, see presence of checkbox
        ![ReportParametersDetails_presence](https://github.com/user-attachments/assets/baa83b55-f5ad-4963-a25c-bcaac5a05f16)
